.changeset/secure-workflows-delete.md

```
@@ -0,0 +1,21 @@
+'@mastra/core': minor
+'@mastra/server': minor
+'@mastra/libsql': patch
+'@mastra/mongodb': patch
+'@mastra/mssql': patch
+'@mastra/mysql': patch
+'@mastra/pg': patch
+'@mastra/spanner': patch
+'@mastra/code-sdk': patch
+---
+
+Added owner-qualified dynamic workflow deletion and same-owner validation for dynamic nested workflows.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)